### ISSUE TITLE
Fix biome language server start on windows

### ISSUE THREAD

**Issue by ramipellumbi** [biomejs/biome-zed#54]
See https://github.com/biomejs/biome-zed/pull/49#issuecomment-2423216359

**Comment by ematipico** [comment]
PRs are welcome 🤗

**Comment by dannya** [comment]
I'm still getting this error on Zed Nightly on Windows:

![Image](https://github.com/user-attachments/assets/1c959add-b841-4f0c-82b5-9ec001f63b76)

I assume this is because a new release of this plugin has not been made since this fix:

![Image](https://github.com/user-attachments/assets/b94c5de9-5bc3-4f02-bf38-1b450ad344de)

@ematipico Please can you release a new version of this plugin?

**Comment by ematipico** [comment]
The latest version **is** 0.1.3. Not sure what you mean

**Comment by dannya** [comment]
@ematipico I would like a new release (0.1.4) with the following fix in:

https://github.com/biomejs/biome-zed/commit/aa9a622e624cbc90ba97758787b16f18e1d84e44

Based on the dates of both the PR (Oct 19, 2024) and version 0.1.3 release (Sep 18, 2024), it seems that this fix is not in a released version.

**Comment by ematipico** [comment]
No, the last commit is the 19th of October, which is 0.1.3, it's right there in the screenshot. Check the dates of the commits.

**Comment by dannya** [comment]
Yes, last commit is 19th of October, but 0.1.3 was released on 18th of September, which is in the screenshot too.

Last commit to the v0.1.3 tag was 18th of September.

**Comment by ematipico** [comment]
Ok, let's wait for @luckydye to provide an answer, he's the person that did the last commit

**Comment by luckydye** [comment]
Currently published is the version from commit https://github.com/biomejs/biome-zed/commit/5878de9243c6ce35ff25e514b3141555d8b16a7e , so 3 commits behind main.

Published commit can be seen here: https://github.com/zed-industries/extensions/tree/main/extensions

Versioning is kind of messy here, since its always published manually with a PR to the zed repository.
